var/spack/repos/builtin/packages/py-graphene-tornado/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGrapheneTornado(PythonPackage):
+    """Graphene Tornado integration."""
+
+    homepage = "https://github.com/graphql-python/graphene-tornado"
+    pypi = "graphene-tornado/graphene-tornado-2.6.1.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("2.6.1", sha256="953bf812267177224ce1ac2a93c669069d85a8fa187a9fac681b76b63dffebc2")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-six@1.10.0:", type=("build", "run"))
+    depends_on("py-graphene@2.1:2", type=("build", "run"), when="@:2")
```

Comment:
Because there is only one version in this package, the `when` statement is not needed here.